fluent/bind/mirror.go

```diff
@@ -0,0 +1,1099 @@
+package bind
```

Comment:
I'd move this to `node/bind` instead of `fluent/bind`.

(And possibly declare it as `package bindnode`, to match theme with `package basicnode`... although if you want to argue that `basicnode` shouldn't have been named like that, yeah, okay, maybe so.)

It's a node implementation moreso than a way of working with other node implementations.  So definitely goes under `node/*` rather than `fluent/*`.

Reply:
Fair enough. I'll put it under node/bindnode though, because `import ".../bindnode"` is always going to be shorter and simpler than `import bindnode ".../bind"`.

Reply:
Done.